.gitignore

```diff
@@ -4,3 +4,5 @@
 .idea
 .DS_Store
 coverage.out
+*__debug_*
+config.json
```

Comment:
Can you set the file ending?

Reply:
It is the executable build when debugging, so it haven't got any extension.